src/randomGamePicker/randomGamePicker.c

```diff
@@ -68,10 +68,12 @@ int main(int argc, char *argv[])
     DIR *dp;
     struct dirent *ep;
     char config_path[512];
-    
+
     if ((dp = opendir("/mnt/SDCARD/Emu")) != NULL) {
-        
+
         while ((ep = readdir(dp))) {
+
+            if (argc > 1 && strcmp(argv[1], ep->d_name) != 0) continue;
```

Comment:
This is inefficient, I think. When you already have the name (`argv[1]`) you don't need to loop through the `Emu` folder, just skip that and set the config path directly.

Reply:
Done! i've created a external method to do that and if we receive an arg it will set directly!